modules/ossm-cert-manager-integration.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: CONCEPT
+[id="ossm-cert-manager-integration_{context}"]
+== Integrating cert-manager with Service Mesh
+
+Cert-manager adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates. 
```

Comment:
Per cert-manager's FAQ (https://cert-manager.io/docs/faq/#how-to-write-cert-manager), cert-manager must always be written in all lower case.

```suggestion
cert-manager adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates. 
```

Reply:
Very good catch @monroe-dylan-m